content/about/guardians.toml

```diff
@@ -1,34 +1,34 @@
 [[guardians]]
 name = "Prof. Jon Crowcroft"
-picture = "/path/to/"
+picture = "/about/crowcroft.jpg"
```

Comment:
I think just the filename would be nice, we can build the full path in the template code (if we even need to, a relative link should be fine).